reference/docs-conceptual/components/ise/How-to-Write-and-Run-Scripts-in-the-Windows-PowerShell-ISE.md

```diff
@@ -149,8 +150,8 @@ Press **CTRL+S** or, on the toolbar, click the **Save** icon, or on the **File**
 
 ### To save a script in ASCII encoding
 
-By default, Windows PowerShell ISE saves new script files (.ps1), script data files (.psd1), and
-script module files (.psm1) as Unicode (BigEndianUnicode) by default.Â To save a script in another
+By default, Windows PowerShell ISE saves new script files (`.ps1`), script data files (`.psd1`), and
+script module files (`.psm`1) as Unicode (BigEndianUnicode) by default.Â To save a script in another
```

Comment:
Looks like the 1 got left outside the codefence.  There's also a weird A on this line.

Reply:
fixed